source/extensions/filters/common/fault/fault_config.cc

```diff
@@ -44,10 +50,10 @@ FaultAbortConfig::FaultAbortConfig(
   }
 }
 
-absl::optional<Http::Code> FaultAbortConfig::HeaderAbortProvider::statusCode(
+absl::optional<Http::Code> FaultAbortConfig::HeaderAbortProvider::httpStatusCode(
     const Http::RequestHeaderMap* request_headers) const {
-  absl::optional<Http::Code> ret;
-  const auto header = request_headers->get(HeaderNames::get().AbortRequest);
+  absl::optional<Http::Code> ret = absl::nullopt;
```

Comment:
I don't think that we need `= absl::nullopt` part here - I think that we should get rid of it if it's not needed.

Reply:
Yep its not required, I put that in there to make it apparent. I had to read `absl::optional` details to find out this, so I thought making it apparent might help readability. I can remove it if its something basic.

Reply:
It's fine either way. I wouldn't worry about it unless we need another push.